src/main/java/com/google/cloud/genomics/dataflow/pipelines/CountReads.java

```diff
@@ -130,9 +141,25 @@ public void processElement(DoFn<Long, String>.ProcessContext c) throws Exception
       }
     }).named("toString"));
     readCountText.apply(TextIO.Write.to(options.getOutput()).named("WriteOutput"));
+
     p.run();
   }
 
+  private static boolean GCSURLExists(String url) {
```

Comment:
This is a generally useful method.  If its not already available in a GCS java library we depend upon, it would be better in `com.google.cloud.genomics.dataflow.util`